airbyte-integrations/connectors/source-klaviyo/integration_tests/sample_state.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Should we use it file if we always have `abnormal_state.json `? It looks like a small duplicate, WDYT?